### ISSUE TITLE
Confuse about the my_layer_norm function and GAN loss function

### ISSUE THREAD

**Issue by SeeU1119** [researchmm/AOT-GAN-for-Inpainting#2]
Hi, thank you for your excellent work, I get a good result when run the demo. But when I read the source code, I get some problems about GAN loss function.
![image](https://user-images.githubusercontent.com/38234791/113799483-bd76f100-9787-11eb-9a55-5af909b1680a.png)
 In the paper, the dis loss about fake_img should  be self.loss_fn(d_fake, gauss(1 - mask)), but I find you just do gauss(mask), Is there something wrong with my understanding？
 whatmore, the dis loss about real_img should be self.loss_fn(d_real, d_real_label), where d_real_label is torch.ones(...), but you write it to torch.zeros(...).
By the way, could you explain the work of my_layer_norm in the AOT block?
Thanks.

**Comment by zjlinkin** [comment]
I have the same confuse about the gan loss part...

**Comment by ewrfcas** [comment]
The same question

**Comment by 964728623** [comment]
d_fake_label = gaussian_blur(masks, (self.ksize, self.ksize), (10, 10)).detach().cuda()
        d_real_label = torch.zeros_like(d_real).cuda()
        # g_fake_label = torch.ones_like(g_fake).cuda()
        g_fake_label = torch.zeros_like(g_fake).cuda()

        dis_loss = self.loss_fn(d_fake[masks>0.5], d_fake_label[masks>0.5]) + self.loss_fn(d_real, d_real_label)
        gen_loss = self.loss_fn(g_fake[masks>0.5], g_fake_label[masks>0.5])
